hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -87,6 +88,9 @@
   private long averageRecordSize = 0;
   private HoodieLogFile currentLogFile;
   private Writer writer;
+  private String filePath = "null";
+  private int logVersion = 0;
```

Comment:
in general., naming these variables with some context with some documentation, would help readability of this file. 

e.g `initialLogVersion` ,  `filePath` -> `logFilePathWritten` etc